catalog/source/workflows.yml

```
@@ -685,6 +685,194 @@ workflows:
+    workflow_description:
+      The workflow for Illumina-sequenced ARTIC data builds on the
+      RNASeq workflow for paired-end data using the same steps for mapping and variant
+      calling, but adds extra logic for trimming ARTIC primer sequences off reads with
+      the ivar package. In addition, this workflow uses ivar also to identify amplicons
+      affected by ARTIC primer-binding site mutations and tries to exclude reads derived
+      from such tainted amplicons when calculating allele-frequencies of other variants.
+    ploidy: HAPLOID
+    taxonomy_id: 694009
+    parameters:
+      - key: Paired Collection
+        type_guide:
+          class: Collection
+          ext:
+            - fastqsanger
+            - fastqsanger.gz
+      - key: NC_045512.2 FASTA sequence of SARS-CoV-2
+        variable: ASSEMBLY_ID
+      - key: ARTIC primer BED
+        type_guide:
+          class: File
+          ext: bed
+      - key: ARTIC primers to amplicon assignments
+        type_guide:
+          class: File
+          ext: tabular
+      - key: Read removal minimum AF
+        type_guide:
+          class: float
+      - key: Read removal maximum AF
+        type_guide:
+          class: float
+      - key: Minimum DP required after amplicon bias correction
+        type_guide:
+          class: integer
+      - key: Minimum DP_ALT required after amplicon bias correction
+        type_guide:
+          class: integer
+    active: true
+  - trs_id: "#workflow/github.com/iwc-workflows/sars-cov-2-pe-illumina-wgs-variant-calling/COVID-19-PE-WGS-ILLUMINA/versions/v0.2.4"
+    categories:
+      - VARIANT_CALLING
+    workflow_name: "COVID-19: variation analysis on WGS PE data"
+    workflow_description:
+      This workflows performs paired end read mapping with bwa-mem
+      followed by sensitive variant calling across a wide range of AFs with lofreq
+    ploidy: HAPLOID
+    taxonomy_id: 694009
+    parameters:
+      - key: Paired Collection
+        type_guide:
+          class: Collection
+          ext:
+            - fastqsanger
+            - fastqsanger.gz
+      - key: NC_045512.2 FASTA sequence of SARS-CoV-2
+        variable: ASSEMBLY_ID
+    active: true
+  - trs_id: "#workflow/github.com/iwc-workflows/sars-cov-2-se-illumina-wgs-variant-calling/COVID-19-SE-WGS-ILLUMINA/versions/v0.1.5"
+    categories:
+      - VARIANT_CALLING
+    workflow_name: "COVID-19: variation analysis on WGS SE data"
+    workflow_description:
+      This workflows performs single end read mapping with bowtie2
+      followed by sensitive variant calling across a wide range of AFs with lofreq
+    ploidy: ANY
+    parameters:
+      - key: Single End Collection
+        type_guide:
+          class: Collection
+          ext:
+            - fastqsanger
+            - fastqsanger.gz
+      - key: NC_045512.2 FASTA sequence of SARS-CoV-2
+        type_guide:
+          class: File
+          ext:
+            - fasta
+            - fasta.gz
```

```suggestion
      - key: NC_045512.2 FASTA sequence of SARS-CoV-2
        variable: ASSEMBLY_FASTA_URL
```
is probably what you would want to do ?

---

Yes, I'm getting there..